fetch.bs

```diff
@@ -5968,14 +5988,25 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
  <li><p>If <var>request</var>'s <a for=request>response tainting</a> is "<code>basic</code>", then
  return success.
 
+ <li><p>Let <var>origin</var> be the result of <a>serializing a request origin</a> with
+ <var>request</var>.
+
+ <li><p>Return the result of calling <a>TAO origin check</a> with <var>origin</var> and
+ <var>response</var>.
+</ol>
+
+<p>To perform a <dfn id=concept-tao-origin-check>TAO origin check</dfn> for an
+<a for=/>origin</a> <var>origin</var> and a <a for=/>response</a> <var>response</var>, run these
+steps:
+
+<ol>
  <li><p>Let <var>values</var> be the result of
  <a for="header list">getting, decoding, and splitting</a> `<code>Timing-Allow-Origin</code>` from
  <var>response</var>'s <a for=response>header list</a>.
 
  <li><p>If <var>values</var> <a for=list>contains</a> "<code>*</code>", then return success.
 
- <li><p>If <var>values</var> <a for=list>contains</a> the result of
- <a>serializing a request origin</a> with <var>request</var>, then return success.
+ <li><p>If <var>values</var> <a for=list>contains</a> <var>origin</var>, then return success.
```

Comment:
If _origin_ is an origin this doesn't work, right? It seems elsewhere you are invoking this with a string, so maybe _origin_ is supposed to be an origin?

Reply:
Changing it to be an ASCII string, as I need to use `serialize origin from request`